src/Tasks/AddToWin32Manifest.cs

```diff
@@ -196,7 +196,7 @@ private ManifestValidationResult ValidateManifest(string? manifestPath, XmlDocum
             {
                 if (!string.Equals(supportedArchitecturesNode.InnerText.Trim(), SupportedArchitectures, StringComparison.OrdinalIgnoreCase))
                 {
-                    Log.LogErrorWithCodeFromResources("AddToWin32Manifest.InvalidValueInSupportedArchitectures", supportedArchitecturesNode.InnerText, manifestPath);
+                    Log.LogErrorWithCodeFromResources(null, manifestPath, 0, 0, 0, 0, "AddToWin32Manifest.InvalidValueInSupportedArchitectures", supportedArchitecturesNode.InnerText);
```

Comment:
Is it possible to get the line number here? That'd be a better UX if so, but it's nontrivial with System.Xml.

Reply:
I will try to handle it separately.